src/utils.ts

```diff
@@ -27,6 +27,7 @@ export const SNAPSHOT_SUBGRAPH_URL = {
   '137':
     'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-polygon',
   '250': 'https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-fantom'
+  '4181': 'https://graph.phiswap.com/subgraphs/name/snapshot-labs/snapshot-phi'
```

Comment:
This subgraph is for delegations, right now subgraphs should be hosted by snapshot-labs on thegraph.com, refer to this https://github.com/snapshot-labs/snapshot-subgraph